x/data/server/query_data.go

```diff
@@ -6,22 +6,20 @@ import (
 	"github.com/cosmos/cosmos-sdk/orm/model/ormlist"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-
+	gogotypes "github.com/gogo/protobuf/types"
```

Comment:
ඞ this import looks sus ඞ

Reply:
https://github.com/regen-network/regen-ledger/pull/970/commits/2f202a60b2799a65b05cf8d09b9d0c3657155062